js/w3c/style.js

```diff
@@ -40,6 +40,9 @@ define(
                         .appendTo($("head"))
                         ;
                 }
+                else if (conf.useExperimentalStyles) {
+                    css = "http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/W3C-" + statStyle;
```

Comment:
> On 27 Nov 2015, at 3:14 AM, Shane McCarron <notifications@github.com> wrote:
> 
> In js/w3c/style.js:
> 
> > @@ -40,6 +40,9 @@ define(
> >                          .appendTo($("head"))
> >                          ;
> >                  }
> > +                else if (conf.useExperimentalStyles) {
> > +                    css = "http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/W3C-" + statStyle;
> Well - that's where the stylesheets are. Should we have done something else?

Yeah, we should have stopped and complained that http is unacceptable in 2015. We should make that a rule going forward.

The project is hosted on Github, so we could ask @fantasai to just serve it from gh-pages (it probably already is, but too lazy to check from my phone:))


> …
> —
> Reply to this email directly or view it on GitHub.
>